include/tvTypes.h

```diff
@@ -238,10 +238,12 @@ typedef enum
     tvColorTemp_WARM,                           //!< Color Temperature is Warm
     tvColorTemp_COLD,                           //!< Color Temperature is Cold
     tvColorTemp_USER,                           //!< User defined color temperature
+    tvColorTemp_SUPERCOLD,                      //!< Color Temperature is Supercold
```

Comment:
the fact that you changed this, kinda means it's not fit for purpose.

What about changing the interface to support k values returned, and then the interface never needs to change again?

Reply:
to make this interface backwards compatible you must only add elements onto the end of the list, so the order of the list does not change... And all the enums are numbered in the previous interface as they are in the current new interface..

Needs to be reviewed by architecture, will check with pete

Reply:
Done.

Reply:
Pete's comment here:
To make this truly extensible we should not use enums.
When colour temperatures where added we never thought that there would be anything more than Normal, Cold, Warm and User. Then we got all the colour temps and now we have super_cold.
 
The fact that it keeps growing suggest that it is something that will potentially grow in the future and we should plan for it.
I would prefer to see a solution close to how we manage picture modes.
It allows the picture modes to grow without changing the interface.
Each picture mode is defined by a "well known name" that the app can translate to the language of choice.
It does not matter about the configuration of the temperature that is a product tuning exercise.
 
But this solution is broken as we will not have boost or super cool for Xumo products and it is unlikely that we will add them.
This is a Llama only solution and therefore should not ship.

Reply:
The open question therefore is, can we do it the way that's being suggested now, rather than adding more technical debit that we'll have to undo?

Reply:
The addition of enums at the end of the declaration is acceptable because it does not change existing enum values and is therefore backward compatible.

It is still Product specific because these names are product specific. However, considering other factors and knowing the impact of change. We can go with the enum solution.

Reply:
please raise an internal ticket to handle the change in the future, but you must comment the code to state this:

/* #TODO: This enum list will be deprecated,  no further extensions can take place, It will be replaced by a picture mode string list, that can be defined by a "well-known name" to allow future scaling. */